templates/documents/general_list.html

```diff
@@ -4,11 +4,14 @@
 {php}$this->assign('GLOBALS', $GLOBALS);{/php}
 
 <link rel="stylesheet" href="{$GLOBALS.css_header}" type="text/css">
+<link rel="stylesheet" href="{php}echo $GLOBALS['assets_static_relative']{/php}/dropzone-4-3-0/dist/dropzone.css">
```

Comment:
@robertdown ,
This is already a smarty script and it's best to not break into pure php within smarty scripts(for example, the new generation of smarty does not allow pure php), if possible. This is a minor issue, so won't push it, though.

Regarding asset path name, definitely feel free to rename; but do it on entire codebase, rather than 1 spot, and keep it a global to ensure scope of it reaches inside classes/methods.